pkg/scalers/apache_kafka_scaler.go

```diff
@@ -0,0 +1,682 @@
+package scalers
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	sigv4 "github.com/aws/aws-sdk-go/aws/signer/v4"
```

Comment:
I recently changed the aws sdk to v2. Can you use aws sdk v2 here?

Reply:
OK got it. Have merge conflicts to resolve anyway.

To be clear #4953 is complete migration to AWS SDK V2 right?

Reply:
yeah, we have changed it totally. Now KEDA uses AWS SDK V2